server/service/src/sync/translations/invoice.rs

```diff
@@ -97,8 +97,7 @@ pub struct LegacyTransactRow {
     #[serde(rename = "currency_ID")]
     #[serde(deserialize_with = "empty_str_as_option_string")]
     pub currency_id: Option<String>,
-    pub currency_rate: Option<f64>,
-
+    pub currency_rate: f64,
```

Comment:
So this one is always set in mSupply?

Reply:
It defaults to 1.0 anyways when created. 🤷🏻 which is the rate for home currencies